lib/scopes/form526_submission_state.rb

```diff
@@ -24,7 +24,9 @@ module Form526SubmissionState
       }
 
       scope :accepted_to_primary_path, lambda {
-        accepted_to_lighthouse_primary_path.or(accepted_to_evss_primary_path)
+        lh = accepted_to_lighthouse_primary_path.pluck(:id)
+        evss = accepted_to_evss_primary_path.pluck(:id)
+        where(id: lh + evss)
```

Comment:
Thanks for doing this! I'm curious about why you didn't just do ` {  accepted_to_lighthouse_primary_path + accepted_to_evss_primary_path}`?

Reply:
so both of those scopes return an activeRecord relation. If you add activerecord relations they are coreced to an array. We want to return an Relation so we can chain it with other active record methods e.g. the timebox

Reply:
🤦🏻‍♀️ yes, of course! I misread and thought you were plucking ids on the final query. this is great. 👍🏻